.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "firmware/ChibiOS"]
 	path = firmware/ChibiOS
-	url = https://github.com/rusefi/ChibiOS.git
-	branch = stable_20.3.x.rusefi
+	url = https://github.com/nmschulte/ChibiOS.git
```

Comment:
Becomes FOME-Tech/ChibiOS  / stable_20.3.x.FOME before (and after ...) this merges.